### DISCUSSION TITLE
Count up total  events (elapsed count per block)

### DISCUSSION THREAD

**Discussion by sebastianferron** [cpvalente/ontime#690] [Ideas]
Hello everyone, a pleasure to greet you, I am a television producer and I am part of an important television network where I live, currently our prime time show is the most watched in our country, not long ago I found this tool that I found fabulous, I started using it. implement with the production team, content and operators in central control and the main stage, however there are many shows where the time is determined by what happens on the main stage and is not structured, there are shows that are a little more improvised and gets carried away by the action of the host and his guests, so time does not become so structured and the elapsed time must be counted and this happens with many shows in different countries, where a lot of work is done with the rating also. For this reason I suggest an option where one can count the elapsed time of a total of events divided by the blocks.
For example, if I have 4 blocks, in each block I have approximately 5 or 6 events (host actions, commercial mentions, videos, games, etc.) I am interested in counting the elapsed time of each block, so that during the next block I can calculate recording or live time, adjust it internally and record or go live less time to comply with the schedule.

The current clock works perfectly for structured events and structured shows such as newscasts, broadcasts of events such as the Oscars, but for entertainment shows where many things happen and everything is very dynamic, it did not adjust and the clock is left aside, unless we can count the elapsed time of a total of block events. We could even have the option of additionally entering the time it should last, to know how much time is left to complete the total time. For example, I have a 60-minute show, and in block 1 I have to record 15 minutes and I record 20, I know that in the next block I have to record 10.

I hope it is of great help and they can implement it in V3.

Greetings to all.

**Comment by cpvalente** [comment]
Hi @sebastianferron , thank you for reaching out. It is great to have users being involved with specification

We are aware of this point of improvement and is something we are actively working on.
The details of the implementation are not yet clear, but the aim is:
- we want a way to track the progress of the runtime against its schedule. so that we always know how much ahead / behind we are #100 
- we want to display to display the time to the next event, accounting for schedule differences #160 
- we want to provide a show reporter which demonstrates which parts ran late or early #165 

With these we hope the user experience of managing delays will be greatly improved.

What do you think?
It would be great to have users input here
